package/Dockerfile

```diff
@@ -0,0 +1,118 @@
+# Builder image
+FROM registry.suse.com/bci/openjdk:17 AS builder
+
+ARG TARGETOS
+ARG TARGETARCH
+
+COPY admin /src/admin
+COPY cli /src/cli
+COPY common /src/common
+COPY images /src/images
+COPY licenses /src/licenses
+COPY java.security /src/java.security
+COPY package /src/package
+COPY project /src/project
+COPY scripts /src/scripts
+COPY build.sbt .scalafix.conf .scalafmt.conf /src/
+
+WORKDIR /src
+
+RUN zypper refresh && \
+    zypper install -y ca-certificates wget curl zip git awk nodejs20 npm20
+
+RUN if [ "$TARGETARCH" = "amd64" ]; then \
+    curl -fL https://github.com/coursier/launchers/raw/c2eb9a75082b4f70b12104b6da74cfff3191dbcd/cs-x86_64-pc-${TARGETOS}.gz | gzip -d > cs; \
+  elif [ "$TARGETARCH" = "arm64" ]; then \
+    curl -fL https://github.com/VirtusLab/coursier-m1/releases/download/v2.1.17/cs-aarch64-pc-${TARGETOS}.gz | gzip -d > cs; \
```

Comment:
To make automation easier (for auto bumps), the version/commit can be extracted to its own variable. However, this is personal preference, feel free to keep as is.